src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -72,11 +72,11 @@ const TreeItem: React.FC<TreeItemProps> = props => {
 
   const getBasename = osPlatform === 'win32' ? path.win32.basename : path.basename;
 
-  const isRoot = fileMap[ROOT_FILE_ENTRY].filePath === treeKey;
```

Comment:
We were quite inconsistent with the tree key as sometimes it ended as the absolute file path and sometimes as <root> (e.g. on reload). For easiest mental model I made the treeKey the same as the fileMap key, basing myself on these docs:

src/models/fileentry.ts:8:
> the path of the file relative to the root folder - used as key in the fileMap

src/models/appstate.ts:82
>  /** maps filePath to FileEntry
>  * - filePath is relative to selected rootFolder
>  * - fileMap[**ROOT_FILE_ENTRY**] is the FileEntry for the rootFolder and it's **filePath is absolute**
>  */

Reply:
You are correct that for `fileMap[ROOT_FILE_ENTRY]` the `filePath` is actually the absolute path of the project.
This root entry should've been a separate property in the AppState but due to the fact that it's heavily used in many places, a refactor would take a lot of time. Maybe we should update the comment on the `fileMap` to reflect this better.

I think it's okay to use the fileMap key as the treeKey, it should be unique